geode-management/src/main/java/org/apache/geode/management/api/RestTemplateClusterManagementServiceTransport.java

```diff
@@ -73,6 +75,27 @@
   public RestTemplateClusterManagementServiceTransport(RestTemplate restTemplate) {
     this.restTemplate = restTemplate;
     this.restTemplate.setErrorHandler(DEFAULT_ERROR_HANDLER);
+
+    // configur the rest template to use a speciic jackson converter
+    List<HttpMessageConverter<?>> messageConverters = restTemplate.getMessageConverters();
+    MappingJackson2HttpMessageConverter jackson2HttpMessageConverter = messageConverters.stream()
+        .filter(MappingJackson2HttpMessageConverter.class::isInstance)
+        .map(MappingJackson2HttpMessageConverter.class::cast)
+        .findFirst().orElse(null);
+
+    if (jackson2HttpMessageConverter == null) {
+      jackson2HttpMessageConverter = new MappingJackson2HttpMessageConverter();
+      messageConverters.add(jackson2HttpMessageConverter);
+    }
+
+    jackson2HttpMessageConverter.setPrettyPrint(false);
+    // the client should use a mapper that would ignore unknown properties in case the server
+    // is a newer version than the client
+    jackson2HttpMessageConverter
+        .setObjectMapper(GeodeJsonMapper.getMapperIgnoringUnknownProperties());
+    // if we don't set the default charset here, the request will use ServletRequest's default
+    // charset which is ISO-8859
```

Comment:
I think this comment is overly specific. If we don't set the charset, the request will use some default, but it may not be ISO-8859. I've been able to change the default on my Mac and in docker images by:

- setting the `LC_ALL` shell variable.
- specifying the `file.encoding` JVM option (which I think overrides `LC_ALL`, but I'm not sure).

Consider removing the comment. To me, `setDefaultCharset(UTF_8)` is clear enough without it.

But if you want a comment, consider: "... which may not be UTF-8."